cf-serverd/cf-serverd-functions.h

```diff
@@ -25,29 +25,11 @@
 #ifndef CFENGINE_CF_SERVERD_FUNCTIONS_H
 #define CFENGINE_CF_SERVERD_FUNCTIONS_H
 
-
-#include <platform.h>
```

Comment:
You need this. `platform.h` should always be first (or a header file that directly includes it first is also OK.)

Reply:
Doesn't really matter, but first line is `generic_agent.h` which includes `cf3.defs.h`.

Reply:
It matters, because the pattern is that you find it included either at the top of your-file.c, or at the top of your-file.h. Before it used to as you say, that it includes some file that includes some file, that includes it at the top, but it was impossible to enforce and subtle breakage happened. Feel free to document this in HACKING.md. ;-)